src/PointwiseFunctions/Hydro/MassWeightedFluidItems.hpp

```diff
@@ -44,6 +46,20 @@ template <typename DataType>
 struct TildeDUnboundUtCriterion : db::SimpleTag {
   using type = Scalar<DataType>;
 };
+
+/// Contains TildeD * (coordinates in frame Fr).
+/// Label allows us to restrict the data to the x>0 (A) or x<0 (B)
+/// plane in grid coordinates (useful for NSNS). Use ObjectLabel::None
+/// to use the entire volume data.
+template <typename DataType, size_t Dim, ::domain::ObjectLabel Label,
+          typename Fr = Frame::Inertial>
+struct MassWeightedCoords : db::SimpleTag {
+  using type = tnsr::I<DataType, Dim, Fr>;
+  ::domain::ObjectLabel object_label = Label;
+  static std::string name() {
+    return "MassWeightedCoords" + domain::name(Label);
```

Comment:
This isn't a change request, but for binaries this naming makes sense; but for single stars, I wouldn't automatically know what `MassWeightedCoordsA` in an input file means. Not sure what to do about this. Is it even helpful to find the CoM of just x>0 for a single star?

Reply:
Not really; I'm not planning to include it in the input files for Grmhd (they'll only have the ObjectLabel::None case), only for GhGrmhd... this is truly working under the assumption that we are working towards binary star systems.

Reply:
Even though you'll only put the None case in the input file, the help string that's printed when you misspell a tensor name with still have the A and B case in it. Do we make a distinction between single star GH-GRMHD and binary GH-GRMHD executables in the metavariables?

Reply:
Agreed; I meant that the pure mhd case (no GH) won't have the A and B case. The GhGrmhd does, even for single stars. But as of now, I do not think we can easily separate the two cases? I don't mind just adding the None case in the executable for now though, the other one isn't useful until we get binaries (I just want to have it available).
I'll also need to add an integral of TildeD over the half plane anyways...

Reply:
I'd say leave the instantiations there, just don't add A and B to the integrals you can observe just yet. Let's wait until we're further along with binaries.